pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -130,3 +130,116 @@ const (
 	MPIImplementationOpenMPI MPIImplementation = "OpenMPI"
 	MPIImplementationIntel   MPIImplementation = "Intel"
 )
+
+// JobStatus represents the current observed state of the training Job.
+type JobStatus struct {
+	// Conditions is an array of current observed job conditions.
+	// +options
```

Comment:
Did you mean `+optional`?

Also add `+listType=Map`

Reply:
> Did you mean +optional?

Oops... Yes, that is `+optional`.

> Also add +listType=Map

Sure.

Reply:
Done.